exposing-to-internet/deploy.sh

```diff
@@ -0,0 +1,154 @@
+
+# Create a sample Apigee environment group and attach the environment
+echo -n "Creating environment group..."
+OPERATION=$(apigeecli envgroups create -o "$PROJECT" -d "$RUNTIME_HOST_ALIAS" -n "$ENVIRONMENT_GROUP_NAME" -t "$TOKEN" | jq --raw-output '.name' | awk -F/ '{print $4}')
+wait_for_operation "$OPERATION"
+
+echo -n "Attaching environment to group..."
+OPERATION=$(apigeecli envgroups attach -o "$PROJECT" -e "$ENVIRONMENT_NAME" -n "$ENVIRONMENT_GROUP_NAME" -t "$TOKEN" | jq --raw-output '.name' | awk -F/ '{print $4}')
+wait_for_operation "$OPERATION"
+
+# Create a Google managed SSL certificate
+echo "Creating SSL certificate..."
+gcloud compute ssl-certificates create sample-apigee-ssl-cert \
+    --domains="$RUNTIME_HOST_ALIAS" --project "$PROJECT" --quiet
+
+## Create a global Load Balancer
+echo "Creating external load balancer..."
+
+# Create a PSC NEG
+gcloud compute network-endpoint-groups create sample-apigee-neg \
+  --network-endpoint-type=private-service-connect \
+  --psc-target-service="$SERVICE_ATTACHMENT" \
+  --region="$REGION" \
+  --network="$NETWORK" \
+  --subnet="$SUBNET" \
+  --project="$PROJECT" --quiet
+
+# Create a backend service and add the NEG
+gcloud compute backend-services create sample-apigee-backend \
+  --load-balancing-scheme=EXTERNAL_MANAGED \
+  --protocol=HTTPS \
+  --global --project="$PROJECT" --quiet
+
+gcloud compute backend-services add-backend sample-apigee-backend \
+  --network-endpoint-group=sample-apigee-neg \
+  --network-endpoint-group-region="$REGION" \
+  --global --project="$PROJECT" --quiet
+
+# Create a Load Balancing URL map
+gcloud compute url-maps create sample-apigee-urlmap \
+  --default-service sample-apigee-backend --project="$PROJECT" --quiet
+
+# Create a Load Balancing target HTTPS proxy
+gcloud compute target-https-proxies create sample-apigee-https-proxy \
+  --url-map sample-apigee-urlmap \
+  --ssl-certificates sample-apigee-ssl-cert --project="$PROJECT" --quiet
+
+# Create a global forwarding rule
+gcloud compute forwarding-rules create sample-apigee-https-lb-rule \
+  --load-balancing-scheme=EXTERNAL_MANAGED \
+  --network-tier=PREMIUM \
+  --address=sample-apigee-vip --global \
+  --target-https-proxy=sample-apigee-https-proxy --ports=443 --project="$PROJECT" --quiet
+
+echo -n "Waiting for certificate provisioning to complete (may take some time)..."
+while true
+do
+  TLS_STATUS="$(gcloud compute ssl-certificates describe sample-apigee-ssl-cert --format=json --project "$PROJECT" --quiet | jq -r '.managed.status')"
+  if [ "$TLS_STATUS" = "ACTIVE" ]; then
+    break
+  fi
+  echo -n .
+  sleep 10
+done
+
+# Pause to allow TLS setup to complete
+sleep 30
+
+echo "Installing dependencies and running tests..."
+npm install
+npm run test
+
+echo "# To send an EXTERNAL test request, execute the following command:"
+echo "curl -v https://$RUNTIME_HOST_ALIAS/healthz/ingress -H 'User-Agent: GoogleHC'"
```

Comment:
Since `$RUNTIME_HOST_ALIAS` is created as part of the script, its not available once the script is finished. Will be nice if we can print the export command as an echo statement

`echo "export RUNTIME_HOST_ALIAS=$RUNTIME_HOST_ALIAS"`